search/retrieval.cpp

```diff
@@ -13,12 +13,13 @@
 #include "indexer/feature_algo.hpp"
 #include "indexer/feature_data.hpp"
 #include "indexer/index.hpp"
-#include "indexer/osm_editor.hpp"
 #include "indexer/scales.hpp"
 #include "indexer/search_delimiters.hpp"
 #include "indexer/search_string_utils.hpp"
 #include "indexer/trie_reader.hpp"
 
+#include "editor/osm_editor.hpp"
```

Comment:
Editor depends on indexer, doesn't it? If so, move this up.

Reply:
done